torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -38,7 +38,8 @@ defmodule Torchx.NxDoctestTest do
         norm_integer: 3,
         all_close?: 3,
         atan2: 2,
-        lu: 2
+        lu: 2,
+        sum: 2
```

Comment:
Sorry, to be clear, we don’t want to exclude sum. Change the doctest to keep only s8-s64 and we should be good. :)

Reply:
That's nice! I was ready to rewrite all `sum/2` doctests in regular form for Torchx:)